Emby.Server.Implementations/Localization/Core/en-US.json

```diff
@@ -75,7 +75,6 @@
     "Songs": "Songs",
     "StartupEmbyServerIsLoading": "Jellyfin Server is loading. Please try again shortly.",
     "SubtitleDownloadFailureFromForItem": "Subtitles failed to download from {0} for {1}",
-    "SubtitlesDownloadedForItem": "Subtitles downloaded for {0}",
```

Comment:
why is this removed?

Reply:
@JustAMan Not used anywhere, simply dead. Probably there are a lot of other strings which are not used, but I found this one only while tinkering with multiple strings.

I plan on check all the dead strings deeply once I get some localization stuff sorted out in Web and everywhere else, do you think it's better to hold this one back to that future PR?